portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+
+async function getQuotes() {
+    const responseFromServer = await fetch('/welcome');
+    const quote = await responseFromServer.json();
+
+    
+
+    const word = quote[Math.floor(Math.random() * quote.length)];
+
+    const welcomeItems = document.getElementById('welcome-container');
+    welcomeItems.innerHTML = word;
```

Comment:
I believe that setting `.innerHTML` can be problematic if `word` contains any HTML characters. Because `word` comes from the server, it is best to code safely on the frontend by assuming it might contain HTML characters which could break your page. I believe that setting `.innerText` instead of `.innerHTML` will automatically ensure that HTML characters get escaped properly so that they are not interpreted as part of the page itself, but only text.